src/com/facebook/buck/apple/AppleDescriptions.java

```diff
@@ -545,7 +546,8 @@ static AppleBundle createAppleBundle(
         Optional<FrameworkDependencies> frameworkDependencies =
             resolver.requireMetadata(
                 BuildTarget.builder(dep)
-                    .addFlavors(FRAMEWORK_FLAVOR)
```

Comment:
We still need this line; this applies to the dependencies and not the bundle itself.